src/tests/global_variables/env_variables_defaults.py

```diff
@@ -60,7 +60,7 @@ class _EnvVariables(DataPool, ABC):
 
     namespace: EnvVar = EnvVar(["NAMESPACE"], default=consts.DEFAULT_NAMESPACE)
     spoke_namespace: str = EnvVar(["SPOKE_NAMESPACE"], default=consts.DEFAULT_SPOKE_NAMESPACE)
-    olm_operators: EnvVar = EnvVar(["OLM_OPERATORS"], loader=lambda operators: operators.lower().split(), default="")
+    olm_operators: EnvVar = EnvVar(["OLM_OPERATORS"], loader=lambda operators: operators.lower().split(","), default=[])
```

Comment:
makes much more sense. thank you!
leaving it to what @adriengentil said, and also let's document it in the README

Reply:
I don't think this is the right approach.
We are holding almost all the env vars in the `_EnvVariables` object, the option to add lambada was added to prevent us from crating a lot of functions just for single call.
I don't see the benefit of creating a new method just for split

Reply:
It's not about creating a new function, it's to make sure the existing one behaves in the same way.
Let's delete that function if it does not make sense (it doesn't seem to be used elsewhere).

Reply:
The main reason that I don't think it should be a separate function is mainly because that I don't want to handle the env vars elsewhere.
It caused us a lot of issues before we had most of them in the same place.

Reply:
@eliorerz not sure I follow you
``parse_olm_operators_from_env`` isn't used anywhere, and it has the older format (``OLM_OPERATORS="odf cnv"``)
what do we gain from keeping it instead of just deleting it?